registry/storage/driver/gcs/gcs.go

```diff
@@ -47,10 +47,13 @@ import (
 const driverName = "gcs"
 const dummyProjectID = "<unknown>"
 
-//DriverParameters A struct that encapsulates all of the driver parameters after all values have been set
+// driverParameters A struct that encapsulates all of the driver parameters after all values have been set
```

Comment:
Doesn't really matter since this won't appear in godoc, but "driverParameters is..." would be more consistent with other comments.